bootstrap-samples/pom.xml

```diff
@@ -51,6 +51,12 @@
             <groupId>de.agilecoders.wicket</groupId>
             <artifactId>wicket-bootstrap-themes</artifactId>
         </dependency>
+        <!-- icons -->
+        <dependency>
```

Comment:
This dependency comes as transitive one from here: https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/c95661ed224909f28342e45ec11f1c71c5b19241/bootstrap-extensions/pom.xml#L61-L65
Just update it from 5.7.1 to 5.9.0 there